web/packages/teleport/src/WorkloadIdentity/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,185 @@
+import { ComponentType, useEffect, useRef, useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import { Box, ButtonPrimary, H1, Image, Text } from 'design';
+import { Theme } from 'design/theme';
+import {
+  DetailsTab,
+  FeatureContainer,
+  FeatureSlider,
+} from 'shared/components/EmptyState/EmptyState';
+
+import { FeatureBox } from 'teleport/components/Layout';
+
+import feature1dark from './workload_identity_feature_1_dark_mode-min.svg';
+import feature1light from './workload_identity_feature_1_light_mode-min.svg';
+import feature2dark from './workload_identity_feature_2_dark_mode-min.svg';
+import feature2light from './workload_identity_feature_2_light_mode-min.svg';
+import feature3dark from './workload_identity_feature_3_dark_mode-min.svg';
+import feature3light from './workload_identity_feature_3_light_mode-min.svg';
+
+const maxWidth = '1204px';
+
+export function EmptyState() {
+  const [currIndex, setCurrIndex] = useState(0);
+
+  const PreviewPanel = tabs[currIndex].PreviewPanel;
+
+  const intervalId = useRef<number>();
+  function handleOnClick(clickedIndex: number) {
+    if (intervalId.current !== null) {
+      clearInterval(intervalId.current); // Clear the interval if it exists
+    }
+    setCurrIndex(clickedIndex);
+  }
+
+  useEffect(() => {
+    intervalId.current = window.setInterval(() => {
+      setCurrIndex(latestIndex => (latestIndex + 1) % tabs.length);
+    }, 5000);
+
+    return () => {
+      if (intervalId.current !== null) {
+        clearInterval(intervalId.current); // Cleanup on unmount
+      }
+    };
+  }, []);
+
+  return (
+    <FeatureBox>
+      <Box mt={4} data-testid="tag-empty-state">
+        <Box mb={3}>
+          <H1 mb={3}>What is Workload Identity</H1>
+          <Text css={{ maxWidth }}>
+            Based on the open-source SPIFFE standard, Workload Identity replaces
+            long-lived API keys and environment secrets that are vulnerable to
+            breaches, mistakes and exfiltration with short-lived cryptographic
+            identities using x.509 certificates or JWTs.
+          </Text>
+        </Box>
+        <FeatureContainer py={2} pr={2}>
+          <Box css={{ position: 'relative' }}>
+            <FeatureSlider $currIndex={currIndex} />
+            {tabs.map((tab, index) => (
+              <DetailsTab
+                key={tab.title}
+                active={currIndex === index}
+                isSliding={!!intervalId}
+                onClick={() => handleOnClick(index)}
+                title={tab.title}
+                description={tab.description}
+              />
+            ))}
+          </Box>
+          <Box mt={-2} height={330}>
+            <PreviewPanel />
+          </Box>
+        </FeatureContainer>
+        {/* setting a max width here to keep it "in the center" with the content above instead of with the screen */}
+        <Box width="100%" maxWidth={maxWidth} textAlign="center" mt={6}>
+          <ButtonPrimary
```

Comment:
did we intentionally want to replace current page or should we add `target: _blank`?